hal/armv7m/imxrt/117x/imxrt.c

```diff
@@ -389,6 +415,481 @@ __attribute__((section(".noxip"))) int _imxrt_setLevelLPCG(int clock, int level)
 }
 
 
+void _imxrt_delay(u32 ticks)
+{
+	/* TODO: use better method e.g. count cycles, must not use IRQs and peripheral timers! */
+	while (ticks-- != 0u) {
+		__asm volatile("nop");
```

Comment:
I am aware of `__asm` vs `__asm__` dispute, let's be consistent for now
```suggestion
		__asm__ volatile ("nop");
```

Reply:
Done.